CHANGELOG.md

```diff
@@ -9,6 +9,7 @@
 
 ## Unreleased
 
+* Add heading option to input component ([PR #1747](https://github.com/alphagov/govuk_publishing_components/pull/1747)) MINOR
```

Comment:
I suggest using 'heading level option' since we already have the heading size option.

```suggestion
* Add heading level option to input component ([PR #1747](https://github.com/alphagov/govuk_publishing_components/pull/1747)) MINOR
```